src/app/shell/header.scss

```diff
@@ -15,10 +15,12 @@
     filter: hue-rotate(160deg) brightness(107%);
   }
 }
-
+body {
+  padding-top: 44px;
```

Comment:
```suggestion
  padding-top: $header_height;
```

Reply:
We have a header height var!? Nice! That’s much better